website/docs/installation/weave-gitops.mdx

```diff
@@ -139,6 +139,8 @@ brew install weaveworks/tap/gitops
 
 :::warning
 This command stores a hash of a password.  While this is relatively safe for demo and testing purposes it is recommend that you look at more secure methods of storing secrets (such as [Flux's SOPS integration](https://fluxcd.io/docs/guides/mozilla-sops/)) for production systems.
+
+More guidance and alternative login methods can be found in [Securing access to the dashboard](../../configuration/securing-access-to-the-dashboard).
```

Comment:
Should this be a single `..` ?

This is `website/docs/installation/weave-gitops.mdx` and the referenced file is in `website/docs/configration/securing-access-to-the-dashboard.mdx` ?

Reply:
I could be misunderstanding how we generate the links when it's deployed tho'

Reply:
Seems to work https://staging.docs.gitops.weave.works/install-update/docs/installation/weave-gitops/#deploying-weave-gitops

Reply:
tried single `..` just to check and got `Page Not Found`